client/a8c-for-agencies/sections/purchases/licenses/license-preview/style.scss

```diff
@@ -151,58 +151,6 @@
 	}
 }
 
-a,
-button {
-	&.license-actions__menu-item {
```

Comment:
I decided to unify the styling for all action menu items, including the one for the Bundle License.